### PR TITLE
Run CI on all branch PRs and pushes

### PR DESCRIPTION
**Pull request by AlmightyYakob**
Previously, CI would only run on PRs and pushes to master.

### CODE REVIEW

**Review by waxlamp — COMMENTED** [review]
Why does this change use a different syntax from the other CI-related change you made recently?

**Comment by AlmightyYakob** [comment]
> Why does this change use a different syntax from the other CI-related change you made recently?

Because we also want to trigger on pushed tags, so I can't just write `on: [push, pull_request]`

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -2,13 +2,9 @@ name: CI
...
 on:
   pull_request:
-    branches:
-      - master
-
   push:
     branches:
-      - master
-
+      - "**"
```
Comment (waxlamp) [inline .github/workflows/main.yml:7]: Can you add a comment explaining why this is needed?

**Review by waxlamp — DISMISSED** [review]
With a comment to explain, this is good by me.

File: .github/workflows/main.yml (hunk 1 above)
Comment (AlmightyYakob) [reply .github/workflows/main.yml:9]: d8bd001

**Review by waxlamp — APPROVED** [review]

**Pull request merged**